### PR TITLE
add options to specify lmin

### PR DESCRIPTION
**Pull request by nleroux3**
A new key was added to the MESH_parameters LVAR_LMIN_STABLE that can be 'CST'/'VAR'/'NON'
If LVAR_LMIN_STABLE = 'CST', the user should specify the variable 'lmin_stable' in the MESH_parameters and that value will be used for sl_Lmin_soil which will be constant in the simulation
If LVAR_LMIN_STABLE = 'VAR', the user should specify the variable 'lmo_winter' in the MESH_parameters and that value will be used to calculate a varying lmin over the winter with the value lmo_winter being the minimum value.
If LVAR_LMIN_STABLE = 'NON', the user does not have to specify another variable and sl_Lmin_soil = -1, i.e. a minimum wind speed is used.

### CODE REVIEW

File: LSS_Model/SVS/svs1/src/runsvs_mesh.F90
Comment (VVionnet) [inline LSS_Model/SVS/svs1/src/runsvs_mesh.F90:1]: @nleroux3 could you remove the comments about SVS2 and SVS1? Indeed, all the LMIN-related options ca ben used with both models.

Comment (nleroux3) [reply LSS_Model/SVS/svs1/src/runsvs_mesh.F90:1]: There is a new commit in which the comments were adjusted

File: Modules/rpnphy/6.1.0/src/surface/sfc_options.F90
Comment (VVionnet) [inline Modules/rpnphy/6.1.0/src/surface/sfc_options.F90:1]: On pourrait utiliser le commentaire suivant pour l'option VAR: 
a variable length of MO is used as a function of the season. It ranges between the value lmo_winter in wintertime (value provided by the user) and 1 in summer.

Comment (nleroux3) [reply Modules/rpnphy/6.1.0/src/surface/sfc_options.F90:1]: Ce commentaire est en effet beaucoup plus clair. C'est change

**Review by VVionnet — APPROVED** [review]

**Pull request merged**